### PR TITLE
HDDS-5158. Add documentation for SCM HA Security.

### PR DESCRIPTION
**Pull request by bharatviswa504**
## What changes were proposed in this pull request?

Add SCM HA security documentation.

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-5158

## How was this patch tested?

Deployed docs locally and verified content.

### CODE REVIEW

File: hadoop-hdds/docs/content/feature/SCM-HA.md [hunk 1]
```diff
@@ -111,6 +107,62 @@ This can be changed with using `ozone.scm.primordial.node.id`. You can define th
 
 Based on the `ozone.scm.primordial.node.id`, the init process will be ignored on the second/third nodes and bootstrap process will be ignored on all nodes except the primordial one.
 
+## SCM HA Security
+
+![Overview](scm-secure-ha.png)
+
+In a secure SCM HA cluster on the SCM where we perform init, we call this SCM as a primordial SCM. 
+Primordial SCM starts root-CA with self signed certificates and is used to issue a signed certificate
```
Comment (xiaoyuyao) [inline hadoop-hdds/docs/content/feature/SCM-HA.md:115]: NIT: self signed  => self-signed

File: hadoop-hdds/docs/content/feature/SCM-HA.md [hunk 2]
```diff
@@ -111,6 +107,62 @@ This can be changed with using `ozone.scm.primordial.node.id`. You can define th
...
+<value>true</value>
+</property>
+
+<property>
+<config>hdds.grpc.tls.enabled</config>
+<value>true</value>
+</property>
+```
+
+Above configs are needed in addition to normal SCM HA configuration.
```
[the remark this answers is not in the document]
Comment (bharatviswa504) [reply hadoop-hdds/docs/content/feature/SCM-HA.md:139]: Done

**Review by bharatviswa504 — COMMENTED** [review]
Thank You @xiaoyuyao for the review. I have addressed review comments.

File: hadoop-hdds/docs/content/feature/SCM-HA.md [hunk 3]
```diff
@@ -111,6 +107,66 @@ This can be changed with using `ozone.scm.primordial.node.id`. You can define th
...
+
+### Primordial SCM:
+
+Primordial SCM is determined from the config ozone.scm.primordial.node.id.
```
Comment (xiaoyuyao) [inline hadoop-hdds/docs/content/feature/SCM-HA.md:144]: Can we add some description about how it is determined when ozone.scm.primordial.node.id is not defined?

Comment (bharatviswa504) [reply hadoop-hdds/docs/content/feature/SCM-HA.md:144]: Updated it

**Review by xiaoyuyao — APPROVED** [review]
LGTM, +1.

**Pull request merged**